packages/appstore/package.json

```diff
@@ -33,6 +33,7 @@
         "@deriv/stores": "^1.0.0",
         "@deriv/trader": "^3.8.0",
         "@deriv/translations": "^1.0.0",
+        "@deriv/hooks": "^1.0.0",
```

Comment:
for this you need to run `npm run bootstrap:dev` to take place on root level package-lock